src/components/common/YourCompany/YourCompany.vue

```diff
@@ -32,8 +32,8 @@
             <div class="company-name font-weight-bold text-uppercase">{{ companyName }}</div>
 
             <!-- Business Type Info -->
-            <template v-if="!hasNewNr && (hasBusinessNameChanged && (isAlterationFiling || isFirmChangeFiling
-              || isFirmConversionFiling))"
+            <template v-if="!hasNewNr && hasBusinessNameChanged && (isAlterationFiling || isFirmChangeFiling ||
+              isFirmConversionFiling)"
```

Comment:
Removed unneeded brackets.
Moved || to previous line (consistent with our other code).